src/main/java/com/crowdin/client/reports/ReportsApi.java

```diff
@@ -51,4 +49,48 @@ public ResponseObject<DownloadLink> downloadReport(Long projectId, String report
         DownloadLinkResponseObject downloadLinkResponseObject = this.httpClient.get(this.url + "/projects/" + projectId + "/reports/" + reportId + "/download", new HttpRequestConfig(), DownloadLinkResponseObject.class);
         return ResponseObject.of(downloadLinkResponseObject.getData());
     }
+
+    /**
+     * @param projectId project identifier
+     * @param limit     maximum number of items to retrieve (default 25)
+     * @param offset    starting offset in the collection (default 0)
+     * @return list of report settings template
+     */
+    public ResponseList<ReportSettingsTemplate> listReportSettingsTemplate (Long projectId, Integer limit, Integer offset) throws HttpException, HttpBadRequestException {
```

Comment:
```suggestion
    public ResponseList<ReportSettingsTemplate> listReportSettingsTemplate(Long projectId, Integer limit, Integer offset) throws HttpException, HttpBadRequestException {
```

Reply:
This space is redundant here. Also, please check the rest of added methods

Reply:
sure.

Reply:
How about now?